### ISSUE TITLE
Daily Task Indicators plugin suggests members activities on login to non-members worlds

### ISSUE THREAD

**Issue by Nightfirecat** [runelite/runelite#1393]
**Java Version and Windows Version**:
```
$ java -version
java version "1.8.0_161"
Java(TM) SE Runtime Environment (build 1.8.0_161-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.161-b12, mixed mode)

$ systeminfo | grep -E '^OS (Name|Version)'
OS Name:                   Microsoft Windows 7 Professional
OS Version:                6.1.7601 Service Pack 1 Build 7601
```

**Reproduction Steps:**
1. Enable the Daily Task Indicators plugin (specifically with the "Show Claimable Staves" option)
2. Log in to a non-members world on a free-to-play account
3. The text "You have staves waiting to be collected from Zaff." will be displayed in chat (see screenshot below)

**Screenshot:**
![Daily Task Indicators plugin incorrectly showing available battlestaves after login on a free-to-play account](https://user-images.githubusercontent.com/2199511/38647147-63881e80-3dda-11e8-9835-18d3147b6cd3.png)

**Comment by Infinitay** [comment]
Fixed in #1129

**Comment by deathbeam** [comment]
This should be fixed in #5212 . Reopen if issue still occurs
